readme.md

```diff
@@ -244,14 +244,13 @@ a single `widget` (`a1b2c3`). What this then implicates is that you may not,
 in general, create a `sprocket` at the root `sprocket` endpoint. The root `sprocket`
 endpoint may list all of the current context's `sprocket`s, or similar, but in
 general should not accept creation requests. In concrete examples:
-  - `POST /sprockets` is invalid.
+  - `POST /sprockets` is invalid unless `widget` is specified in the payload.
```

Comment:
Was initially hesitant to accept this but I think it's alright. If this is done in practice maybe we can settle on a request format that guides the implementation more.